kratos/tests/cpp_tests/utilities/test_divide_triangle_3d_3.cpp

```diff
@@ -0,0 +1,202 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:     BSD License
+//  			 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+//
```

Comment:
Header format.

Reply:
(other than the name) what's wrong with the format? I copied yours from the divide_2d

Reply:
Indentations (B of BSD and K of Kratos must be aligned to first letter of the author). Most probably it is wrong in the base one too.